base/lnls-get-n-unpack.sh

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env bash
+#
+# Download and extract tarball archive from the network.
+
+set -eu
+
+case "$1" in
+    -r) dest=/ ;;
+    -l) dest=. ;;
+    *) >&2 echo "Invalid extraction mode: must be either root (-r) or local (-l)."
+       exit 1;
+    ;;
+esac
+
+shift
+
+for url; do
+    download_dir=$(mktemp -d)
+
+    echo "Downloading $(basename $url)..."
+    wget $url -P $download_dir &> /dev/null
```

Comment:
I'd quote both instances of `$url` here, just to be sure :)

Reply:
Do `wget -P $download_dir -o $download_dir/wget-log  "$url" || (cat $download_dir/wget-log && false)`

Reply:
Good idea! Thanks for the suggestion.